packages/core/lib/commands/migrate/meta.js

```diff
@@ -118,8 +122,18 @@ module.exports = {
         option: "--save",
         description: "Specify whether the migration will save on chain",
         hidden: true
+      },
+      {
+        option: "--url",
+        description:
+          "Creates a provider using the given url and connects to the network."
+      },
+      {
+        option: "--network",
```

Comment:
I'm curious about why you prefer `--network` lives here vs with the other global options.

Actually, can `url` be a global option as well?

Reply:
The global options generally appends the options at the end of the usage. The reason behind keeping `--network` and `--url` options here rather than in the global options, is to show that they are mutually exclusive.

Reply:
Okie. thank you for the explanation.

Reply:
should we add a note in each of these letting the user know that these options are mutually exclusive? do you think that would be helpful?

Reply:
`Usage` section shows that they are mutually exclusive but if adding a note in the descriptions is preferable, then we can add it here too.

Reply:
whatever you think, I don't feel strongly about it :)